### PR TITLE
Kay/database endpoints

### PR DESCRIPTION
**Pull request by kdcgovender**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -2,12 +2,19 @@ import express from 'express';
...
 
 configDotenv();
 const app = express();
 
+app.use(cors({
+  origin: 'http://localhost:3000', 
+  methods: ['GET', 'POST', 'PUT', 'DELETE'],
+  allowedHeaders: ['Content-Type', 'Authorization'] 
+}));
+
```
Comment (seanvanwyk-bbd) [inline server/index.js]: If you've added this cors stuff, you've then left my manual cors header below it, which you can probably merge (for the development check) and delete.

File: client/src/routes/mardown-editor/index.js [hunk 2]
```diff
@@ -31,9 +32,34 @@ Start using Great Note today and unlock a world of possibilities for organizing
...
+    } catch (error) {
+      console.error('Error saving note:', error);
+    }
+  };
+
   return (
     <main id="editor-page">
-      <section className="">
+      <section className="editor-section">
+        <button className="save-button" onClick={handleSave}>Save</button>
```
Comment (seanvanwyk-bbd) [inline client/src/routes/mardown-editor/index.js]: and just remember that Arinaho has added a save button so you're gonna have merge conflicts once everything goes into main. Probably our first item to look at when we get together, is getting main up to date again.

**Review by seanvanwyk-bbd — CHANGES_REQUESTED** [review]
From my side, just check the cors. Can't vouch for all the endpoints working out right now, but I trust they work well!

File: client/src/routes/mardown-editor/index.js (hunk 2 above)
Comment (kdcgovender) [reply client/src/routes/mardown-editor/index.js]: Ah yes. ill remove it in the conflict resolver

**Review by seanvanwyk-bbd — APPROVED** [review]

**Review by arinaho-muleba — APPROVED** [review]

**Pull request merged**